src/tests/png_tests.rs

```diff
@@ -89,12 +89,17 @@ mod tests {
 
         chunk_bytes.append(&mut bad_chunk);
 
-        let png = Png::try_from(chunk_bytes.as_ref());
+        let bytes: Vec<u8> = Png::STANDARD_HEADER
+            .iter()
+            .chain(chunk_bytes.iter())
+            .copied()
+            .collect();
```

Comment:
Same as all the other comments. This is giving away iterator methods.

Reply:
This is copied from [line 46](https://github.com/jrdngr/pngme_book/pull/16/commits/605e0a3068c1548791e7b2e00b7e192a701b6576#diff-64bac96df54b7643fb0586abb21e47f98e339750f483f3d23b18325f75897958L46) elsewhere in the file.